src/app/cient-list/cient-list.component.ts

```diff
@@ -0,0 +1,128 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { WorkerappComponent } from '../workerapp/workerapp.component';
+import { UtilService } from '../util.service';
+import { Cust } from '../Cust';
+import { WorkerAppService } from '../services/worker-app.service';
+import { Planning, Customer, Address, Invoice } from '../interfaces/worker-interfaces';
+
+@Component({
+  selector: 'app-cient-list',
+  templateUrl: './cient-list.component.html',
+  styleUrls: ['./cient-list.component.css']
+})
+export class CientListComponent implements OnInit{
+  @Input() parent?: WorkerappComponent;
+  custArr : Array<Cust> = [];
+  planningList: Planning[] = [];
+  invoicesList: Invoice[] = [];
+  today: Date = new Date();
+  tempCustomer: Customer;
+  tempAddress: Address;
+  address: string;
+  name: string;
+  employeeID: number;
+  tempCust: Cust;
+
+  constructor(private cData: UtilService, private service: WorkerAppService) 
+  {
+    this.tempCustomer = 
+    {
+      CustomerID: 0,
+      FirstName: '',
+      LastName: '',
+      BirthDate: this.today,
+      AdressID: 0,
+      Email: '',
+      PhoneNumber: '',
+      Password: '',
+      GasType: 0,
+      Electricitytype: 0
+
+    };
+
+    this.tempAddress = 
+    {
+      AdressID: 0,
+      City: '',
+      Street: '',
+      HouseNumber: '',
+      PostalCode: '',
+      Country: '',
+      StartDate: this.today,
+      EndDate: this.today  
+
+    };
+
+    this.address = '';
+    this.name = '';
+    this.employeeID = service.eid;
+    this.tempCust = new Cust(this.employeeID);
+  }
+
+  ngOnInit(): void
+  {
+
+    // Get planning
+    this.service.getPlanning()
+    .subscribe(plannings => 
+      { this.planningList = plannings as Planning[]; })
+    
+    // Add each customer to the list
+    this.planningList.forEach(planning => 
+      {
+        if (planning.Date == this.today)
```

Comment:
Will this every be true? Maybe check day instead. Not sure what you are trying to do here.
Also use {} for the if, unless you actually intent to only do the first statement.